doc/cabal-commands.rst

```diff
@@ -977,7 +977,8 @@ Run ``cabal check`` in the folder where your ``.cabal`` package file is.
     Set verbosity level (0–3, default is 1).
 
 ``cabal check`` mimics Hackage's requirements: if no error or warning
-is reported, Hackage should accept your package.
+is reported, Hackage should accept your package. If ``cabal check`` exits
+with ``1``, Hackage will refuse it.
```

Comment:
There's still a grey area but stressed in the text: could it be that errors (i assume, not) or warnings (maybe?) are reported but check record with 0? If it can what does it mean? Anyway, i think it'd be best to address this case explicitly in the manual.

Reply:
Indeed it was not clear. New text:

---

``cabal check`` mimics Hackage's requirements: if no error or warning
is reported, Hackage should accept your package. If errors are present      
``cabal check`` exits with ``1`` and Hackage will will refuse the package.

---

Reply:
It does, thank you!

Sorry, there was a typo and i meant to create a suggestion but accidentally committed it...